app/shared/src/GestureController.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+
+type State = {
+  key: string,
+  isTransitioning: boolean,
+  index: number,
+  routes: mixed[],
+};
+
+import { NativeModules } from 'react-native';
+
+export default function GestureController(
```

Comment:
As I said, instead of a function, make it a class or an object with appropriate methods.